StardewNewsFeed/Services/GameService.cs

```diff
@@ -67,6 +69,19 @@ public class GameService : IGameService {
                 }
             }
         }
+
+        public void CheckSilos() {
+            var farm = Game1.getFarm();
+            var silos = farm.buildings.Where(b => b.buildingType == new Netcode.NetString("Silo"));
+            var maxCapacity = silos.Count() * 240.0;
+            var piecesOfHay = farm.piecesOfHay;
+            var percentOfMax = piecesOfHay / maxCapacity;
```

Comment:
Check for capacity. If they don't have any silos, we're looking at a divide-by-zero exception.

Reply:
Good catch! A commit addressing this has been pushed